modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import cv2 as cv
+import glob
+import argparse
+
+def init_tracker(tracker_name):
+    if tracker_name == "Boosting":
+        tracker = cv.TrackerBoosting_create()
+        frames_before_reinit = 500
+    elif tracker_name == "MIL":
+        tracker = cv.TrackerMIL_create()
+        frames_before_reinit = 1000
+    elif tracker_name == "KCF":
+        tracker = cv.TrackerKCF_create()
+        frames_before_reinit = 110
+    elif tracker_name == "MedianFlow":
+        tracker = cv.TrackerMedianFlow_create()
+        frames_before_reinit = 20
+    elif tracker_name == "GOTURN":
+        tracker = cv.TrackerGOTURN_create()
+        frames_before_reinit = 20
+    elif tracker_name == "MOSSE":
+        tracker = cv.TrackerMOSSE_create()
+        frames_before_reinit = 20
+    elif tracker_name == "CSRT":
+        tracker = cv.TrackerCSRT_create()
+        frames_before_reinit = 250
+    return [tracker, frames_before_reinit]
+
+def main():
+    parser = argparse.ArgumentParser(description="Run LaSOT-based benchmark for visual object trackers")
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="Test part of LaSOT dataset", help="Full path to LaSOT folder")
+    args = parser.parse_args()
+
+    list_of_videos = [None for _ in range(280)]
+    file_with_video_names = open(args.path_to_dataset + "/testing_set.txt", "r")
+
+    for i in range(len(list_of_videos)):
+        list_of_videos[i] = file_with_video_names.readline()
+
+    trackers = {'Boosting': 500, 'MIL': 1000, 'KCF': 110,
+                'MedianFlow': 20, 'GOTURN': 20, 'MOSSE': 20, 'CSRT': 250}
```

Comment:
Duplicated logic with number of iterations?

Reply:
Do you mean equivalent numbers for different trackers? If yes, all numbers were received via experiments with video sequence.

Reply:
I mean that there are already these numbers in the `init_tracker` method above

Reply:
I just forgot to remove dictionary `trackers` after adding `init_tracker` method, my bad.
Here is enough to use list for trackers loop, like you mentioned earlier: 
`
trackers = [
  "Boosting",
  "MIL",
  "KCF",
  "MedianFlow",
  "GOTURN",
  "MOSSE",
  "CSRT"
]
`